specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2024-09-01-preview/redisenterprise.json

```diff
@@ -881,14 +953,257 @@
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "RedisEnterprise",
+          "Databases",
+          "AccessPolicyAssignments"
+        ],
+        "operationId": "AccessPolicyAssignment_Delete",
+        "description": "Deletes a single access policy assignment.",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "x-ms-examples": {
+          "RedisEnterpriseAccessPolicyAssignmentDelete": {
+            "$ref": "./examples/RedisEnterpriseAccessPolicyAssignmentDelete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DatabaseNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AccessPolicyAssignmentNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The access policy assignment delete operation started for database. GET the URL in the Azure-AsyncOperation or Location headers to retrieve updated status.",
+            "headers": {
+              "Location": {
+                "description": "Location URI to poll for result",
+                "type": "string",
+                "format": "uri"
+              },
+              "Azure-AsyncOperation": {
+                "description": "Azure-AsyncOperation URI to poll for result",
+                "type": "string",
+                "format": "uri"
+              }
+            }
+          },
+          "204": {
+            "description": "Successfully deleted the access policy assignment."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/redisEnterprise/{clusterName}/databases/{databaseName}/accessPolicyAssignments": {
+      "get": {
+        "tags": [
+          "RedisEnterprise",
+          "Databases",
+          "AccessPolicyAssignments"
+        ],
+        "operationId": "AccessPolicyAssignment_List",
+        "description": "Gets all databases in the specified Redis Enterprise cluster.",
```

Comment:
This description is inaccurate. Copy/paste issue?

Reply:
corrected. thanks